test/mocks/providers/provider_circular_value.dart

```diff
@@ -1,15 +1,19 @@
 import "package:flutter/material.dart";
 import "package:hooks_riverpod/hooks_riverpod.dart";
 import "package:proxima/views/components/async/circular_value.dart";
+import "package:proxima/views/helpers/types.dart";
 
-ProviderScope circularValueProvider(AsyncValue<void> value) {
+ProviderScope circularValueProvider(
+  Future<int> Function() future, [
+  Widget Function(BuildContext, Object)? fallbackBuilder,
+]) {
   return ProviderScope(
     child: MaterialApp(
       home: Scaffold(
         body: CircularValue(
-          value: value,
+          future: future().mapRes(),
           builder: (context, data) => const Text("Completed"),
-          fallbackBuilder: (context, error) => const Text("Strange Error"),
+          fallbackBuilder: fallbackBuilder ?? CircularValue.defaultFallback,
         ),
       ),
```

Comment:
Small suggestion, since you are adding the `fallbackBuilder` shouldn't we also add the `builder` for completeness ?

Reply:
Sure, not sure why we should do it now if nobody needs it at the moment, but I added it in for completeness. 893a9d0c

I'm more in favor of adding those things when they are needed rather than upfront, because more often than not, nobody will need them.